documentation/book/master.adoc

```diff
@@ -9,7 +9,7 @@ This guide describes how to install and use {ProductLongName}.
 
 Apache Kafka is a popular platform for streaming data delivery and processing. For more details about Apache Kafka
 itself visit http://kafka.apache.org[Apache Kafka website]. The aim of {ProductName} is to make it easy to run
-Apache Kafka on {ProductPlatformName}.
+Apache Kafka on {ProductPlatformName}. {ProductName} is based on Apache Kafka 1.1.0.
```

Comment:
Sentence per line.
Shouldn't we say "{ProductName} {ProductVersion} is based on Apache Kafka 1.1.0", just to be explicit? And maybe it's worth using "{KafkaVersion}", just so it's easier to update (it might only be used once in the whole document, but you have to find it first).

Reply:
Sentence per line should be fixed.

> Shouldn't we say "{ProductName} {ProductVersion} is based on Apache Kafka 1.1.0", just to be explicit?

I think that would look strange in master ... _Strimzi master is based on Apache Kafka 1.1.0._ And for releases the version is usually somewhere in the document header to make it clear enough which version do we talk about. WDYT?

> And maybe it's worth using "{KafkaVersion}", just so it's easier to update (it might only be used once in the whole document, but you have to find it first).

I thought about this a lot already when writing it. I think we can now afford to support only one version of Kafka because we are in early stage. But soon enough we will need to start thinking about supporting multiple versions ... if only to offer upgrade path etc. So it seemed that I will create the variable for it just to remove it later when we have to change the docu due to multiple versions support. But I can add it if you insist of course, it is not a big change.